src/gui/usbipd_gui.rs

```diff
@@ -25,6 +26,7 @@ pub(super) trait GuiTab {
 #[derive(Default, NwgUi)]
 pub struct UsbipdGui {
     device_notification: Cell<DeviceNotification>,
+    menu_tray_event_handler: RefCell<Option<nwg::EventHandler>>,
```

Comment:
You could use `Cell` instead of `RefCell` to avoid having to borrow stuff around, since you only need to `Cell:set` when creating the event handler, and either use `Cell:take` or `Cell:update` when unbinding the handler.

Reply:
This is great, I'd love to avoid RefCell!